.github/workflows/interop.yml

```diff
@@ -80,12 +80,17 @@ jobs:
           # TODO(#1366)
           cp interop_client/docker/test-runner/main.go.patch mls-implementations/interop/test-runner
           cd mls-implementations/interop
+          go mod edit -go=1.18
```

Comment:
1.18 is pretty old (2022). Maybe updating to something recent solves the issue?

Reply:
1.22 is the most recent and worked for me locally